Dokan/Dokan/Scenes/ProductDetails/Cell/ReviewTableViewCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  reviewTableViewCell.swift
+//  Dokan
+//
+//  Created by raniazeid on 20/07/2022.
+//
+
+import UIKit
+
+class ReviewTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var reviewerImageView: UIImageView!
```

Comment:
We mark our `@IBOutlet`s and `@IBAction`s with `private` better practice to be a habit 👍